src/libs/PolicyUtils.ts

```diff
@@ -197,6 +197,26 @@ function isPendingDeletePolicy(policy: OnyxEntry<Policy>): boolean {
     return policy?.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE;
 }
 
+/**
+ * Checks if the current user has a shared policy.
+ */
+function hasSharedPolicies(policies: OnyxCollection<Policy>, policyMembersCollection: OnyxCollection<PolicyMembers>): boolean {
+    let found = false;
```

Comment:
NAB: I think we should rename it to something more specific like `hasSharedPoliciesFlag`.

Reply:
@rayane-djouah Sorry for the confusion. I was taking about the `found` variable not the function name. `hasSharedPolicies` is fine for function name.